tests/test_client.py

```diff
@@ -1201,6 +1201,21 @@ def test_exchange_success(self):
         self.assertEqual('dummy_revoke_uri', credentials.revoke_uri)
         self.assertEqual(set(['foo']), credentials.scopes)
 
+    def test_exchange_success_binary_code(self):
+        payload = (b'{'
+                   b'  "access_token":"SlAV32hkKG",'
```

Comment:
Yes guilty as charged, I used the test_exchange_test as the basis for this test. Could you point me to a good example test?